Agda.cabal

```
@@ -158,24 +156,23 @@ library
     buildable: False
 
   build-depends:  aeson >= 0.11.3.0 && < 1.5
-                , array >= 0.5.1.0 && < 0.6
+                , array >= 0.5.1.1 && < 0.6
                 , async >= 2.2 && < 2.3
-                , base >= 4.8.0.0 && < 4.13
-                , binary >= 0.7.3.0 && < 0.9
+                , base >= 4.9.0.0 && < 4.13
+                , binary >= 0.8.3.0 && < 0.9
                 , blaze-html >= 0.8 && < 0.10
                 , boxes >= 0.1.3 && < 0.2
-                , bytestring >= 0.10.6.0 && < 0.11
-                , containers >= 0.5.6.2 && < 0.7
+                , bytestring >= 0.10.8.1 && < 0.11
+                , containers >= 0.5.7.1 && < 0.7
                 , data-hash >= 0.2.0.0 && < 0.3
-                , deepseq >= 1.4.1.1 && < 1.5
-                , directory >= 1.2.2.0 && < 1.4
-                -- EdisonCore 1.3.2 doesn't compile with GHC 7.10.*.
-                , EdisonCore == 1.3.1.1 || >= 1.3.2.1 && < 1.4
+                , deepseq >= 1.4.2.0 && < 1.5
+                , directory >= 1.2.6.2 && < 1.4
+                , EdisonCore == >= 1.3.1.1 && < 1.4
```

Here's the parse error Travis is complaining about. I guess you want
```suggestion
                , EdisonCore >= 1.3.1.1 && < 1.4
```

---

Ah, thank you!